log/util.go

```diff
@@ -0,0 +1,57 @@
+package log
+
+import (
+	"fmt"
+	"reflect"
+)
+
+// InterleavedKVToFields converts keyValues a la Span.LogKV() to a Field slice
+// a la Span.LogFields().
+func InterleavedKVToFields(keyValues ...interface{}) ([]Field, error) {
+	if len(keyValues)%2 != 0 {
+		return nil, fmt.Errorf("Non-even keyValues len: %v", len(keyValues))
```

Comment:
"uneven" doesn't refer to even-odd-ness... more like "unlevelness".

![image](https://cloud.githubusercontent.com/assets/2251553/18611468/e7f70eb4-7cee-11e6-9420-c14ef2eade7e.png)